src/gui/EQ/EQEditor.cpp

```diff
@@ -74,5 +135,7 @@ void EQEditor::resized()
     drawButton.setBounds (linearPhaseButton.getBoundsInParent().translated (-pad - buttonDim, 0));
     drawCheckButton.setBounds (drawButton.getBoundsInParent());
     drawXButton.setBounds (linearPhaseButton.getBoundsInParent());
+    spectrumAnalyser.setBounds (getLocalBounds());
+    spectrumAnalyser.toBack();
```

Comment:
Having a `toBack()` seems a little odd... what happens if we don't do this, or if we call `toBack()` in the constructor?